install-deps.sh

```diff
@@ -23,6 +23,10 @@ export LC_ALL=en_US.UTF-8 # the following is vulnerable to i18n
 
 ARCH=$(uname -m)
 
+function in_jenkins() {
+    test -n "$JENKINS_HOME"
```

Comment:
looks good to me, just a optional refactor suggestion to combine ( test -n "$JENKINS_HOME") && "CI DEBUG: $@"

Reply:
I'm not following.  Like...

```
function in_jenkins() {
    test -n "$JENKINS_HOME" && echo "CI_DEBUG: $@"
}

in_jenkins "This is a debug message"
```

???

Reply:
yep, looks fine this way as well.

Reply:
I get where this suggestion comes from, but it misses the fact that this functionality is already in use in this PR to do something other than `echo`. If we ever want to do or log something if we are _not_ in Jenkins, it'd also be an issue